Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/task/TaskActionListener.java

```diff
@@ -26,7 +26,7 @@ public void save(Optional<String> title, Optional<String> description) {
                     .title(title)
                     .description(description)
                     .build();
-            tasksService.save(task).call();
+            tasksService.save(task);
```

Comment:
We don't need Action as return value for service actions anymore since the service handles concurrency.